spring-boot/admin/src/main/java/eu/coatrack/admin/controllers/AdminController.java

```diff
@@ -122,6 +125,38 @@
         chartColorsPerHttpResponseCode = Collections.unmodifiableMap(colorMap);
     }
 
+    private static final int oneMinuteToMilliseconds = 60000;
+    private static final int criticalThresholdInMinutes = 60;
+    private static final int warningThresholdInMinutes = 5;
+
+    private String gatewayHealthStatusSummary(){
+        List<Proxy> proxiesList = proxyRepository.findAvailable();
+        if (proxiesList.stream().map(Proxy::getStatus).anyMatch(status -> status == ProxyStates.CRITICAL))
+            return "Critical";
+        else if(proxiesList.stream().map(Proxy::getStatus).anyMatch(status -> status == ProxyStates.WARNING))
+            return "Warning";
+        return "Ok";
```

Comment:
Would be better to use enum values here instead of hard-coded strings.

Reply:
Done